functional/measured-boot-swtpm-sanity/main.fmf

```diff
@@ -6,7 +6,7 @@ description: |
  Uses accept-all measured boot policy for the verifier.
  Registers agent providing empty policy in --mb_refstate, confirms successful verification.
  Configure python agent to read custom bios_binary_measurements from /var/tmp.
- Re-registers agent providing --mb_refstate generated using create_mb_refstate.
+ Re-registers agent providing --mb_refstate generated using create_mb_refstate or keylime-policy.
```

Comment:
Hi @ansasaki , I think you have mixed two tests. We have `measured-boot-policy-sanity` which you are modifying above and which requires an actual measured boot (UEFI at least) and therefore is not run in keylime CI and there is also `measured-boot-swtpm-sanity` which uses faked `binary_bios_measurements` and is run in CI .. and whose `main.fmf` you are modifying here. I would say that changes you did (using both scripts) make sense primarily for the `swtpm` variant which gets run in CI. For the other test I think we can also use both scripts (since you already did the change).

Reply:
My bad! :sweat_smile: 
I'll make the changes to both tests.

Reply:
Should be fixed now